utils/installer/install.sh

```diff
@@ -1,225 +1,191 @@
+    local CMD="git -C $LUNARVIM_RUNTIME_DIR/lvim pull origin $LVBRANCH"
+    echo "$CMD"
+    eval "$CMD"
+  else
+    clone_lvim "$@"
+    __wait_indicator
+  fi
+
+  echo "Thank you for installing LunarVim!!"
+  echo "I recommend you also install and activate a font \
+    from here: https://github.com/ryanoasis/nerd-fonts"
+}
+
+function detect_platform() {
+  OS="$(uname -s)"
+  if [ "$OS" == "Linux" ]; then
+    grep -q Ubuntu /etc/os-release && RECOMMEND_INSTALL="sudo apt install -y " && return
+    [ -f "/etc/arch-release" ] && RECOMMEND_INSTALL="sudo pacman -s " && return
+    [ -f "/etc/artix-release" ] && RECOMMEND_INSTALL="sudo dnf install -y" && return
+    [ -f "/etc/fedora-release" ] && RECOMMEND_INSTALL="sudo dnf install -y " && return
+    [ -f "/etc/gentoo-release" ] && RECOMMEND_INSTALL="emerge install -y " && return
+  elif [ "$OS" == "Darwin" ]; then
+    RECOMMEND_INSTALL="brew install"
+  else
+    echo "OS $OS is not currently supported."
+    exit 1
+  fi
+}
+
+function check_dep() {
+  if ! command -v "$1" &>/dev/null; then
+    echo "Missing dependency! \
+        Please try the following command and re-run the installer"
+    echo "$RECOMMEND_INSTALL $1"
+    exit 1
+  fi
+}
+
+function install_npm_deps() {
+  check_dep "npm"
+  check_dep "yarn"
+  echo "Installing with npm.."
+  for dep in "${__npm_deps[@]}"; do
+    if ! npm ls -g "$dep" &>/dev/null; then
+      echo "installing $dep .."
+      npm install -g "$dep"
+    fi
+  done
+  echo "all nodejs dependencies are succesfully installed"
+}
+
+function install_pip_deps() {
+  check_dep "pip3"
+  echo "Installing with pip.."
+  for dep in "${__pip_deps[@]}"; do
+    pip3 install --user "$dep"
+  done
+  echo "all python dependencies are succesfully installed"
+}
+
+function backup_old_config() {
+  for dir in "${__lvim_dirs[@]}"; do
+    # we create an empty folder for subsequent commands \
+    # that require an existing directory
+    mkdir -p "$dir" "$dir.bak"
+    if command -v rsync &>/dev/null; then
+      rsync --archive -hh --partial --info=stats1 --info=progress2 \
+        --modify-window=1 "$dir" "$dir.bak"
+    else
+      cp -R "$dir/*" "$dir.bak/."
+    fi
+  done
+}
+
+function install_packer() {
+  git clone --depth 1 https://github.com/wbthomason/packer.nvim \
+    "$LUNARVIM_RUNTIME_DIR/site/pack/packer/start/packer.nvim"
+}
+
+function clone_lvim() {
+  echo "Backing up old LunarVim configuration"
+  backup_old_config
+  echo "Cloning LunarVim configuration"
+  case "$@" in
+    *--testing*)
+      cp -r "$(pwd)" "$LUNARVIM_RUNTIME_DIR/lvim"
+      ;;
+    *)
+      local CLONE_CMD="git clone https://github.com/$LV_REPO --branch $LVBRANCH --depth 1 \
+        $LUNARVIM_RUNTIME_DIR/lvim"
+      echo "$CLONE_CMD"
+      eval "$CLONE_CMD"
+      ;;
+  esac
+
+  if [ -f "$LUNARVIM_RUNTIME_DIR/lvim/utils/bin/lvim" ]; then
+    echo "Install LunarVim shim"
+    cp "$LUNARVIM_RUNTIME_DIR/lvim/utils/bin/lvim" "$HOME/.local/bin"
```

Comment:
Can we expect that the user had this path in the PATH?

Reply:
So I'm relying on the fact that people should have this, since it's used by `pip` for example. 

I'm also considering removing this all together and use a shell function and append it to the user bashrc/zshrc. But then we'd need to ask the user every time.

Reply:
I think the current one in this PR is the lesser evil. Changing user configs are gonna be iffy at best, imo.